app/src/main/res/layout/app_version_activity.xml

```diff
@@ -29,13 +29,24 @@
       app:titleTextColor="@color/white" />
   </com.google.android.material.appbar.AppBarLayout>
 
+  <View
+    android:id="@+id/app_version_shadow_view"
+    android:layout_width="0dp"
+    android:layout_height="4dp"
+    android:layout_gravity="top"
+    android:background="@drawable/toolbar_drop_shadow"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintTop_toBottomOf="@id/app_version_app_bar_layout"
+    app:layout_constraintBottom_toTopOf="@id/app_version_fragment_placeholder"/>
```

Comment:
Remove this last constraint. It is not required. 
If the shadow does not appear on top after these cases then just shift the entire `View` tag below `FrameLayout`

Reply:
Done